src/pandas_profiling/version.py

```diff
@@ -1,2 +1,2 @@
 """This file is auto-generated by setup.py, please do not alter."""
-__version__ = "3.3.0"
+__version__ = "3.3.1"
```

Comment:
There's something wrong with the PR, you are "dragging" files not expected. 

@JamesTook can you please check if you've rebased correctly?

Reply:
FWIW, this change happened to me in PR #1076 too, but I just decided not to stage it. I'm not sure which automagical process alters this...